paasta_tools/cli/cmds/spark_run.py

```diff
@@ -331,16 +308,6 @@ def add_subparser(subparsers):
         default=CLUSTER_MANAGER_K8S,
     )
 
-    list_parser.add_argument(
-        "--enable-dra",
-        help=(
-            "[DEPRECATED] Enable Dynamic Resource Allocation (DRA) for the Spark job as documented in (y/spark-dra)."
-            "DRA is enabled by default now. This config is a no-op operation and recommended to be removed."
-        ),
-        action="store_true",
-        default=False,
-    )
-
     k8s_target_cluster_type_group = list_parser.add_mutually_exclusive_group()
     k8s_target_cluster_type_group.add_argument(
         "--force-use-eks",
```

Comment:
Remove this

Reply:
And also --force-no-use-eks

Reply:
Verify if not used and remove: 
--suppress-clusterman-metrics, --executor-memory, --executor-cores, --max-cores, --driver-memory, --driver-cores, --driver-max-result-size

Reply:
The above args have been marked as deprecated already, Will remove them in a future release